### PR TITLE
[EDU-1630] - Spaces Member Location example

### PR DESCRIPTION
**Pull request by GregHolmes**
This PR contains the two code samples for the new Examples section we are creating. It contains a Typescript and a React version for Component Locking with Spaces. Which will be rendered in the code sandbox live examples in the future, along with others.

The README within each directory /examples/spaces-member-location/javascript and /examples/spaces-member-location/react contains information on how to get these running.

[Jira epic ticket](https://ably.atlassian.net/browse/EDU-1541)
[Jira ticket](https://ably.atlassian.net/browse/EDU-1630).

### CODE REVIEW

**Review by ttypic — APPROVED** [review]
LGTM

File: examples/spaces-member-location/page.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+Member location are implemented using [Ably Spaces](https://ably.com/docs/products/spaces). The Spaces SDK contains a set of purpose-built APIs that abstract away the complexities involved in managing the locations of participants focus in a collaborative application. It is built on top of Ably's core platform, and so it provides the same performance and scaling guarantees.
+
+## Resources
+
+// React brief (Only visible if viewing the React example)
+
+Use the following components to add member location into an application:
+
+* [`SpacesProvider`](https://ably.com/docs/spaces/react#spaces-provider): initializes and manages a shared space client instance, passing it down through React context to enable real-time spaces functionality across the application.
+* [`SpaceProvider`](https://ably.com/docs/spaces/react#spaces-provider): manages the state and functionality of a specific space, providing access to real-time interactions within that space via React context.
```
Comment (m-hulbert) [inline examples/spaces-member-location/page.md:16]: real-time --> realtime

File: examples/spaces-member-location/page.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
+# Member location for applications
+
+Track the locations of other users in an application.
```
Comment (m-hulbert) [inline examples/spaces-member-location/page.md:3]:
```suggestion
Track the location of other users within an application.
```

File: examples/spaces-member-location/page.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+
+Track and display the location of users within an application, such as a form field they have selected, the cell they're currently editing in a spreadsheet, or the slide they're viewing on a slide deck.
```
Comment (m-hulbert) [inline examples/spaces-member-location/page.md:5]:
```suggestion
Track and display the location of users within an application, such as which form field they have selected, the cell they're currently editing in a spreadsheet, or the slide they're viewing on a slide deck.
```

File: examples/spaces-member-location/page.md [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+
+Member location are implemented using [Ably Spaces](https://ably.com/docs/products/spaces). The Spaces SDK contains a set of purpose-built APIs that abstract away the complexities involved in managing the locations of participants focus in a collaborative application. It is built on top of Ably's core platform, and so it provides the same performance and scaling guarantees.
```
Comment (m-hulbert) [inline examples/spaces-member-location/page.md:7]:
```suggestion
Member locations are implemented using [Ably Spaces](https://ably.com/docs/products/spaces). The Spaces SDK contains a set of purpose-built APIs that abstract away the complexities involved in managing the locations of participants focus in a collaborative application. It is built on top of Ably's core platform, and so it provides the same performance and scaling guarantees.
```

**Review by m-hulbert — CHANGES_REQUESTED** [review]
Same minor issue as with avatars - I don't think we should be able to have the same colour or name appear for each user (only replicated in JS so far)

Then just a couple of minor copy changes 👍

**Review by m-hulbert — APPROVED** [review]
LGTM!

**Pull request merged**